components/Report/Cards.tsx

```diff
@@ -0,0 +1,75 @@
+import {
+  Accordion,
+  AccordionButton,
+  AccordionIcon,
+  AccordionItem,
+  AccordionPanel,
+  Box,
+  HStack,
+  StackDivider
+} from '@chakra-ui/react';
+import React, { Fragment } from 'react';
+import RangeSection from './RangeSection';
+import dummy from '@/utils/constants/dummy';
+import ResultInfo from '@/utils/types/ResultInfo';
+
+const header = {
+  good: '#E4FCEE',
+  moderate: '#FFDEC1',
+  risk: '#FFE2E2'
+};
+
+interface CardProps {
+  data: ResultInfo;
+}
+
+const Card: React.FC<CardProps> = props => {
+  const { title, resultTab, tipsTab, result } = props.data;
+
+  return (
+    <Accordion allowToggle boxShadow={'xl'} borderRadius={'4'}>
+      <AccordionItem>
+        <h1>
```

Comment:
Why are u wrapping AccordionButton in h1?

Reply:
Good point. It doesn't make a difference. Came from the docs.